stock_package_info/models/product_product.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# Copyright 2015 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2015 AvanzOsc
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    weight_net = fields.Float(
+        string='Net Weight',
+        related='product_tmpl_id.weight_net',
+        help='Weight of contents excluding package weight',
+    )
+
+    @api.multi
+    def write(self, vals):
+        weight_net = vals.get('weight_net')
+        should_cascade = vals.pop('no_cascade', False)
```

Comment:
I'm open to suggestions here as this seems a bit hacky to me. This is my fix for the infinite recursion caused by the fact that the given product is itself a variant of its variants.